src/main/resources/application.properties

```diff
@@ -7,7 +7,10 @@ spring.datasource.password=${USER_PROFILE_DB_PASSWORD:ccd}
 spring.jpa.hibernate.ddl-auto=none
 spring.datasource.type=com.zaxxer.hikari.HikariDataSource
 spring.datasource.hikari.maximum-pool-size=${USER_PROFILE_DB_MAX_POOL_SIZE:2}
+
+liquibase.enabled=${ENABLE_DB_MIGRATE:true}
 liquibase.change-log=classpath:/db/changelog/db.changelog-master.xml
```

Comment:
liquibase.change-log is also specified in the liquibase.properties. Is it also needed here?

Reply:
it is for local development where the app instance is still responsible for updating the schema